go/src/lib.rs

```diff
@@ -543,3 +544,96 @@ pub unsafe extern "C" fn command(
         }
     });
 }
+
+/// Executes a command request received from Go through the FFI interface.
+///
+/// # Arguments
+///
+/// * `client_adapter_ptr` - Pointer to the ClientAdapter instance
+/// * `channel` - Channel identifier for callback routing
+/// * `request_bytes` - Pointer to protobuf-encoded command request bytes
+/// * `request_len` - Length of the request bytes array
+///
+/// # Safety
+///
+#[no_mangle]
+pub unsafe extern "C" fn send_command_request(
```

Comment:
If I understand correctly, this method is only performing the function of `replace_connection_password`. In that case, can we please change the naming and functionality for this? I don't see a use of returning `CommandResponse` from here.

Reply:
@janhavigupta007 This is the correct approach. There are many other commands that are not implemented, and this covers them all.

https://github.com/valkey-io/valkey-glide/blob/7ff4494465c68b1eaa63bf73ba5e35d170060d86/glide-core/src/protobuf/command_request.proto#L516-L528

Reply:
If you check the `command` function, it is already doing one of the functionality from `CommandRequest`. In my opinion, either we should make `command` function generic or use separate dedicated functions for each of the cases in `CommandRequest`.

@eifrah-aws Please share your opinion on the rust side implementation.